common/client-libs/validator-client/src/nymd/mod.rs

```diff
@@ -773,6 +773,31 @@ impl<C> NymdClient<C> {
             .await
     }
 
+    /// Update the configuration of a mixnode. Right now, only possible for profit margin.
```

Comment:
So since you can only update profit margin, perhaps the operation itself should be called `UpdateProfitMargin`?

Reply:
I've named it like that so that we can update other stuff as well in the future, so it should be more generic then just profit margin.